src/openzaak/import_data/tests/utils.py

```diff
@@ -0,0 +1,57 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2019 - 2022 Dimpact
+import csv
+import shutil
+from pathlib import Path
+from unittest import TestCase
+
+from django.conf import settings
+
+from celery.utils.text import StringIO
+
+from openzaak.utils.fields import get_default_path
+
+
+class ImportTestFileMixin(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.remove_upload_files()
+        self.remove_import_files()
+
+    def remove_import_files(self):
+        import_dir = Path(settings.IMPORT_DOCUMENTEN_BASE_DIR)
+        files = import_dir.glob("*")
+
+        for file in files:
+            if file.is_file():
+                file.unlink()
+            else:
+                shutil.rmtree(file)
+
+    def remove_upload_files(self):
+        from openzaak.components.documenten.models import EnkelvoudigInformatieObject
+
+        upload_dir = get_default_path(EnkelvoudigInformatieObject.inhoud.field)
```

Comment:
Are we sure that this is always set? I just want to avoid having `upload_dir` being `/` or `.` and deleting everything in those directories :p